cou/apps/base.py

```diff
@@ -370,16 +372,8 @@ def channel_codename(self) -> OpenStackRelease:
         :return: OpenStackRelease object
         :rtype: OpenStackRelease
         """
-        try:
```

Comment:
I was not able to find why coverage wasn't complaining about this before. We are already checking if channel is valid, so I was not able to write a unit tests that would trigger this `ValueError` and that is why I'm removing it